### ISSUE TITLE
$REPO_NAME no longer includes username/repo

### ISSUE THREAD

**Issue by guycreate** [Shippable/support#2363]
When cloning from a github repository, building and then pushing, I find the new build YML format no longer sets $REPO_NAME to the full github slug (username/repo) instead is just the name.

Before new build system:
$REPO_NAME = Shippable/support 

Now
$REPO_NAME = support

I've had to hard-code the username in my [shippable.yml](https://github.com/iopa-io/iopa-static/blob/master/shippable.yml) but if there is a better way please let me know.   The documentation is a little lacking on all the available environment variables and the Node.js example project is still using the old build format file even though linked from the new documentation.

**Comment by manishas** [comment]
@pranaypareek  we need to make sure we're setting this correctly.

**Comment by pranaypareek** [comment]
@manishas sure, I'm looking into this.

@guycreate we'll keep you posted. Thanks for reaching out!

**Comment by pranaypareek** [comment]
@guycreate we also have another env var (called `ORG_NAME`) which will give you the name of the organization/user that owns the repo. This will save you from hard-coding the name.

In the example of `Shippable/support`:
`$ORG_NAME` is set to `Shippable`
(We'll update the docs with a description for this env var)

To make things easier, we have decided to include another env var called `REPO_FULL_NAME` which will include both the name of the organization/user and the repository.

eg. `$REPO_FULL_NAME = Shippable/support`

We have pushed this to beta. I'll update you when we get this live in production!
